index.html

```diff
@@ -4,22 +4,30 @@
 <div class="row">
   <div class="col-sm-6 content">
     <p>
-      VuFind® is a library resource portal designed and developed for libraries by libraries.
-      The goal of VuFind® is to enable your users to search and browse through all of your
-      library's resources by replacing the traditional OPAC to include:
-      </p><ul>
-        <li>Catalog Records</li>
-        <li>Locally Cached Journals</li>
-        <li>Digital Library Items</li>
-        <li>Institutional Repository</li>
-        <li>Institutional Bibliography</li>
-        <li>Other Library Collections and Resources</li>
-      </ul>
-    <p></p>
+      VuFind® is a discovery system designed and developed for libraries by libraries. It is also flexible enough
+      to build search interfaces for all kinds of content beyond the library environment. The goal of VuFind® is
+      to enable your users to search and browse through all of your resources in a single consistent and user-friendly
+      interface. Content could include:
+    </p>
+    <ul>
+      <li>Catalog records</li>
+      <li>Institutional repository content</li>
+      <li>Open access journal articles</li>
+      <li>Digitized library materials</li>
+      <li>Websites</li>
+      <li>Items available for interlibrary loan</li>
+      <li>Licensed content (where supported by providers)</li>
+      <li>Other collections and resources &mdash; just add metadata!</li>
+    </ul>
     <p>
-      VuFind® is completely modular so you can implement just the basic system,
-      or all of the components. And since it's open source, you can modify the modules to
-      best fit your need or you can add new modules to extend your resource offerings.
+      VuFind® is completely modular and includes tools to support a wide range of use cases, from a basic Library
+      catalog search to a complex "Bento Box" incorporating data from many sources. It supports building a local
```

Comment:
Good point, @dltj. I wanted to avoid making this more verbose, so I opted to change `a complex "Bento Box" incorporating` to `a sophisticated dashboard of`. I think the "dashboard" language is more familiar, and this speaks more to VuFind's flexibility and less to any specific feature (which I think makes it a more future-proof description).

Reply:
Also, "complex" sounds bad but "sophisticated" sounds good. ;-)